docs/operator-guides/refreshes.md

```diff
@@ -0,0 +1,128 @@
+| ------------------------------ | --------------------------------- | --------------------------------- | --------- |
+| Incremental \| Append + Dedupe | Available                         | Available                         | Available |
+| Full Refresh \| Overwrite      | Not available                     | Not available                     | Available |
+| Full Refresh \| Append         | Not available                     | Not available                     | Available |
+| Incremental \| Append          | Not available                     | Available                         | Available |
+
+For supported destinations, and depending on the sync mode you have selected, there are two types of refreshes - "retaining history" and "removing history". Both kinds of Refresh Syncs can be triggered from the UI and via the Airbyte API at either the stream or connection levels (although both might not be available depending on your sync mode).
+
+![Are you sure you want to refresh data](../.gitbook/assets/refreshes/are-you-sure-refresh.png)
+
+## Refresh and Remove Records
+
+Refresh and Remove Records Syncs will only display data in the destination’s final tables that was newly synced since the refresh was initiated. This is the simplest type of refresh, and leads to table mirroring, if you are using an overwrite sync mode.
+
+:::warning
+
+Not all sources keep their history forever. If you perform a Refresh and Remove Records Sync, and your source does not retain all of its records, this may lead to data loss.
+
+:::
+
+## Refresh and Retain Records
+
+Refresh and Retain Records Syncs keep the records you had previously, and merge in the new data from this sync onward with the data previously stored in your destination. Using a Refresh and Retain Records Sync with a Deduplicated Sync mode is the best way to recover from an unreliable source.
+
+:::warning
+
+Refresh and Retain Records Syncs will lead to increased data storage in your destination (as the old and new copies of your data are both stored). This may also lead to increased compute costs and slower syncs, depending on your destination.
+
+:::
+
+## Resumability
+
+Airbyte strives to create resilient syncs that can handle many types of issues (for example, networking issues, flaky source APIs or under provisioned destination servers). Whenever possible, Airbyte syncs are resumable, including standard Full Refresh + Overwrite and Full Refresh + Append syncs, along with the Refreshes we are describing on this page. “Resumable” in this context means that if something goes wrong in the first attempt of your sync, we will immediately try again with a subsequent attempt - in fact, Airbyte will keep retrying as long as the source is producing new records and the destination can commit them. In order to achieve this resumability, Airbyte may deliver a record more than once - Airbyte is a deliver at-least-once system.
```

Comment:
This will be available soon after refreshes. Do we want to make it explicit in the docs for now?

Reply:
Is this different from what the admonation at the top says (This page describes a coming-soon feature. Refreshes will be available as of Airbyte v1.0.0.)?

Reply:
They both should be available by 1.0.0. But technically, since we will be releasing Refresh before RFR, it could be confusing for a user.